### ISSUE TITLE
AdaBoost Classifier/Regressor converter predictions don't match with Scikit predictions for non-default learning rate

### ISSUE THREAD

**Issue by prabhat00155** [onnx/sklearn-onnx#171]
data = load_diabetes()
X = data.data
y = data.target

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)
model = AdaBoostRegressor(base_estimator=None, n_estimators=5, learning_rate=1.2,
                          loss='exponential', random_state=42).fit(X_train, y_train)

onnx_model = convert_sklearn(model, 'lr', [('input', FloatTensorType([1, X_test.shape[1]]))])
save_model(onnx_model, 'lr.onnx')
sess = InferenceSession('lr.onnx')
res = []
for x in X_test:
    res.append(sess.run(None, input_feed={'input': x.astype(np.float32)}))

print(np.mean(np.isclose(model.predict(X_test), list(map(lambda x: x[0][0][0], res)), rtol=1e-2)))

X, y = make_regression(n_features=100, n_samples=10000)

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)
model = AdaBoostRegressor(base_estimator=None, n_estimators=42, learning_rate=1.1,
                          loss='exponential', random_state=42).fit(X_train, y_train)

onnx_model = convert_sklearn(model, 'lr', [('input', FloatTensorType([1, X_test.shape[1]]))])
save_model(onnx_model, 'lr.onnx')
sess = InferenceSession('lr.onnx')
res = []
for x in X_test:
    res.append(sess.run(None, input_feed={'input': x.astype(np.float32)}))

print(np.mean(np.isclose(model.predict(X_test), list(map(lambda x: x[0][0][0], res)))))

X, y = make_regression(n_features=10, n_samples=1000, random_state=42)
X = X.astype(np.int64)

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)
model = AdaBoostRegressor(base_estimator=None, n_estimators=10, learning_rate=0.9,
                          loss='exponential', random_state=42).fit(X_train, y_train)

onnx_model = convert_sklearn(model, 'lr', [('input', FloatTensorType([1, X_test.shape[1]]))])
save_model(onnx_model, 'lr.onnx')
sess = InferenceSession('lr.onnx')
res = []
for x in X_test:
    res.append(sess.run(None, input_feed={'input': x.astype(np.float32)}))

print(np.mean(np.isclose(model.predict(X_test), list(map(lambda x: x[0][0][0], res)))))

output:
0.0
0.0
0.0

**Comment by prabhat00155** [comment]
https://github.com/onnx/sklearn-onnx/pull/223
